docs/import-assistant.md

```diff
@@ -0,0 +1,67 @@
+ # Import Assistant
+ 
+Run powerful AI-enabled commands to assist with your import script development.
+
+&nbsp;
+### Install
+Add an npm script entry to your Edge Delivery project's `package.json`:
+
+```
+"assistant": "aem-import-helper assistant"
+```
+
+### Options
+
+- `--url`: The URL of the page to analyze
+- `--outputPath`: The directory to save the import scripts
+- `--name`: The name of the block
+- `--prompt`: Descriptive text to help understand the content
+
+
+&nbsp;
```

Comment:
? nbsp shouldn't be necessary should it?

Reply:
There appear to be multiple `&nbsp;` that could be removed.

Reply:
In my experience, it was the only way to add a new line that actually worked.  Without them, there's no real separation of command sections.

Reply:
I think we should leave the page formatting up to GitHub. Effective use of Headings should be enough to understand sections.